packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -0,0 +1,190 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { BoxelInput } from '@cardstack/boxel-ui';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { trackedFunction } from 'ember-resources/util/function';
+import type { CardDef } from 'https://cardstack.com/base/card-api';
+import type CardService from '../../services/card-service';
+import { cardTypeDisplayName } from '@cardstack/runtime-common';
+import { or, not } from '@cardstack/boxel-ui/helpers/truth-helpers';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    url: URL;
+    onURLChange: (url: URL) => void;
```

Comment:
so it looks like this callback is invoked when the URL changes. you should add the logic to clear the error state in this callback so that as soon as a user starts typing after an error the error message disappears